next/lib/zod/article.ts

```diff
@@ -14,6 +14,7 @@ export const ArticleBaseSchema = z.object({
   }),
   title: z.string(),
   field_image: ImageShape,
+  field_excerpt: z.string().optional(),
```

Comment:
I added the excerpt to the zod type for the article, so I could use it in the new article-listing-item component.